client/driver/logging/syslog_server_unix_test.go

```diff
@@ -0,0 +1,28 @@
+package logging
+
+import (
+	"testing"
+	"time"
+)
+
+func TestSyslogServer_Start_Shutdown(t *testing.T) {
+	sc := NewSyslogCollector(nil)
+
+	l, err := sc.getListener(20000, 22000)
```

Comment:
Thank you for the comment. As SyslogServer uses unix domain socket on the unix system, I updated it to Dial to the socket for the test.